sql_export_excel/models/sql_export.py

```diff
@@ -69,20 +71,16 @@ def check_column_position(self):
                     _("The column position can't be less than 1.")
                 )
 
-    @api.multi
     def _get_file_extension(self):
         self.ensure_one()
         if self.file_format == "excel":
             return "xlsx"
         else:
             return super()._get_file_extension()
 
-    @api.multi
     def excel_get_data_from_query(self, variable_dict):
         self.ensure_one()
-        res = self._execute_sql_request(
-            params=variable_dict, mode="fetchall", header=self.header
```

Comment:
It is not ok to remove the header parameter here.
If you does it, the header field become useless, but more importante, the whole feature allowing to insert data in an existing xls file (with a lot of formulas for instance) does not work anymore.

This header feature need a modification in `sql_request_abstract` module, that were added here : https://github.com/OCA/server-tools/pull/1659/files

I guess, these modifications are not in version 14, because the port (to v13) was initiated before the PR in v12 was merged.
I believe, it should be added back in this PR to avoid loosing the feature.

Reply:
Hello, @florian-dacosta 
As you know the header parameter is removed from the dependent module, 
can I change it in the dependent module, or can you please suggest to me another good way to add a header feature?

Reply:
You should add it in the dependent module.
It does not really have been removed from dependent module, it is kind on an omission during the migration, so it should be added back. You can do this in this PR IMHO

Reply:
suggestions are implemented can you please re-review it